sdk/python/kfp/compiler/_data_passing_using_volume.py

```diff
@@ -0,0 +1,200 @@
+#/bin/env python3
+
```

Comment:
Thanks Alexy. 

I thought user already can survive with existing SDK (option-1, pipelinevolume, option-2 volume/volumeMount). User's codes can pass string typed volume based file path cross components.

I will setup a meeting with you to quickly sync on this one. I may didn't well catch the reason on why need this PR due to my less knowledge on SDK.